app/api/event_copyright.py

```diff
@@ -40,22 +40,33 @@ class EventCopyrightList(ResourceList):
 
     def query(self, view_kwargs):
```

Comment:
@SaptakS The changes are the same for event-copyright, #3861 also adds permissions. I'll keep this PR to just speakers-call.